packages/terra-compact-interactive-list/src/CompactInteractiveList.jsx

```diff
@@ -70,20 +85,38 @@ const propTypes = {
   width: PropTypes.string,
 
   /**
-   * Columns minimum width should be a valid css string in value in px, em, or rem units..
+   * Callback function that is called when a selectable cell is selected. Parameters:
+   * @param {string} rowId rowId
+   * @param {string} columnId columnId
+   */
+  onCellSelect: PropTypes.func,
+
+  /**
+   * Callback function that is called when all selected cells need to be unselected. Parameters: none.
+   */
+  onClearSelection: PropTypes.func,
+
+  /**
+   * Columns minimum width should be a valid css string in value in px, em, or rem units.
    */
   columnMinimumWidth: PropTypes.string,
 
   /**
-   * Columns maximum width should be a valid css string in value in px, em, or rem units..
+   * Columns maximum width should be a valid css string in value in px, em, or rem units.
    */
   columnMaximumWidth: PropTypes.string,
```

Comment:
Wouldn't the minimum and maximum widths be part of the columnShape?  I would expect to be able to control the values for each column.

Reply:
There are 2 ways to control column max and min width:
Option 1: specify min and max width for a specific column as a part of column shape prop: https://github.com/cerner/terra-framework/blob/466e8c8925bbcc558f73fbe0983801049ef68aa2/packages/terra-compact-interactive-list/src/proptypes/columnShape.js#L27
Option 2: (the one you are looking at) columnMaximumWidth and olumnMinimumWidth prop of the whole component ,this way they'll be applied to each column.

Option 1 takes precedence on Option 2, so if there is columnMaximumWidth set to the whole component and hence applied to all columns, but then some columns have minimumWidth (or maximumWidth) props set for these columns directly, the last ones would be applied to those columns.